### DISCUSSION TITLE
How does the new Multi-object Cable Terminations work?

### DISCUSSION THREAD

**Discussion by PaulR282** [netbox-community/netbox#9792] [Q&A] [answered]
I just installed the beta version to test it. But how exactly can I create cables with multiple terminations? 
E.g. I go to an interface and click "Connect Cable" -> "Front Port". The normal menu shows up like always, where I can select 1 termination.  

![image](https://user-images.githubusercontent.com/47193860/179945561-5d02c2d0-73d7-4b75-9bb9-bdf48c76015b.png)

**Comment by DanSheps** [comment, marked as answer]
In the name field, you keep selecting the items you want.

**Reply by PaulR282** [reply]
That doesn't work, but it's because of #9808. Didn't see I'm still on v3.3-dev
